ci/checks/enforce-correct-shebangs.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+if grep -r -e '^#!\/usr\/bin\/env python$' -e '^#!\/usr\/bin\/env python2$' ../../git_machete; then
+  echo "Ambigous python shebang"
```

Comment:
`Ambiguous`; also, as a rule, include the desired form in the error message so that the developer knows how to address the error